# Q3

The bench hold-down clamp is being used to clamp two boards together while they are being glued as shown in Figure Q1(c). The clamp consists of a screw having single square threads of M12 X 2 and coefficient of friction in the screw thread is taken as 0.2.  The screw producing maximum power of 35 W when moving at 2mm/s axially.

 (i)   the torque acting at the screw 

(ii) the compression force exerted on the boards when torque as (i) is applied  

(iii) Recommend the suitable handle length, d if the compression force is increased to 1.5 kN, thus the user need to apply force of 100 N on the handle. 

In [7]:
# Required
from math import sin, cos, tan, pi, atan, asin, degrees, radians

In [2]:
screw_type = 1 # single-start
alpha = 0
d_outer = 12
pitch = 2
f_screw = 0.2
screw_power = 35
v_linear = 2
print('Power Screw Type: Square thread')

Power Screw Type: Square thread


(i)   the torque acting at the screw

Power is given, we can find the torque using

$$P=\omega T$$
$$P=\frac {2\pi n_{rpm}}{60} T$$

Find $n_{rpm}$ using screw Linear Velocity

$$v_{screw}=Lead\times \frac{n_{rpm}}{60}$$

In [ ]:
Lead = screw_type*pitch
print(f'Lead={Lead}')

Lead, L=2


$$2=2\times \frac{n_{rpm}}{60}$$

In [4]:
n_rpm = 60
print(f'n_rpm={n_rpm}rpm')

n_rpm=60rpm


$$35=\frac {2\pi (60)}{60} T$$

In [6]:
Torque = screw_power*60/(2*pi*n_rpm)
print(f'Torque, T = {Torque:.3f}Nm')

Torque, T = 5.570Nm


(ii) the compression force exerted on the boards when torque as (i) is applied

Free body diagram

$$+\circlearrowleft \sum M_B=0$$
$$-F_A(80)+F_C(300)=0$$

Find force at power screw to find $F_c$
Force at power screw is load, W

$$T_u = \frac{Wd_m}{2} \frac{f+\cos\alpha_n \tan\lambda}{\cos\alpha_n + f\tan\lambda} + \frac{Wf_c d_c}{2}$$

No collar friction and square thread is used

$f_c = 0$ and $\alpha=0 , \alpha_n=0$

$$T_u = \frac{Wd_m}{2} \frac{f+\cos(0) \tan\lambda}{\cos(0) + f\tan\lambda}$$

Find mean diameter, $d_m = \frac {D+(D-p)}{2}$

$d_m = \frac {12+(12-2)}{2}$

In [9]:
d_inner = d_outer-pitch
d_mean = (d_outer+d_inner)/2

print(f'Mean diameter={d_mean}mm')

Mean diameter=11.0mm


From equation
$$ \tan\lambda = \frac{L}{\pi d_m} $$

In [10]:
tan_lambda = Lead/(pi*d_mean)
print(f'tan(lambda)={tan_lambda}')
lambda_helix = degrees(atan(tan_lambda))
print(f'Lambda = {lambda_helix}')

tan(lambda)=0.057874524760689224
Lambda = 3.312271200129051


$$5.57 = \frac{F_A (0.011)}{2} \frac{f+\cos(0) \tan\lambda}{\cos(0) + f\tan\lambda}$$

In [8]:
theta = degrees(asin(100/300))
print(f'theta={theta}')

theta=19.47122063449069
